packages/debug/src/browser/debug-session.tsx

```diff
@@ -93,6 +95,9 @@ export class DebugSession implements CompositeTreeElement {
         readonly id: string,
         readonly options: DebugConfigurationSessionOptions,
         readonly parentSession: DebugSession | undefined,
+        testService: TestService,
```

Comment:
I'm curious about consistency here (beginner question): Why not putting this as a private - or protected - readonly property instead of leaving it as a parameter? I see it is only use in constructor, so that may be the reason? Or to avoid the `override` from the `PluginDebugSession`?

Reply:
Yes, if something is only used in the constructor, no need to add state to the class. State is bad!